appengine/metadata_server/app.rb

```diff
@@ -18,14 +18,14 @@
 
 get "/" do
   uri = URI.parse(
-    "http://metadata.google.internal/computeMetadata/v1" +
+    "http://metadata.google.internal/computeMetadata/v1" \
```

Comment:
I find it easier to understand `+` than `\`, but I'm open to either just wanted to surface it as an opinion.

Reply:
I also much prefer `+`, particularly for website samples.

Using `\` is a fine convention for application/production code, if folks really care about concatenations occurring. But I would like to + `+` for website samples.